src/cryptonote_core/service_node_rules.cpp

```diff
@@ -18,11 +18,19 @@ uint64_t get_staking_requirement(cryptonote::network_type m_nettype, uint64_t he
   if (height < hardfork_height) height = hardfork_height;
 
   uint64_t height_adjusted = height - hardfork_height;
-  uint64_t base = 10000 * COIN;
-  uint64_t variable = (35000.0 * COIN) / loki::exp2(height_adjusted/129600.0);
-  uint64_t linear_up = (uint64_t)(5 * COIN * height / 2592) + 8000 * COIN;
-  uint64_t flat = 15000 * COIN;
-  return std::max(base + variable, height < 3628800 ? linear_up : flat);
+  if (height >= 235987)
+  {
+    uint64_t base     = 15000 * COIN;
+    uint64_t variable = (24721.0 * COIN) / loki::exp2(height_adjusted/129600.0);
+    return base + variable;
+  }
+  else
+  {
+    uint64_t base      = 10000 * COIN;
+    uint64_t variable  = (35000.0 * COIN) / loki::exp2(height_adjusted/129600.0);
+    uint64_t linear_up = (uint64_t)(5 * COIN * height / 2592) + 8000 * COIN;
+    return std::max(base + variable, linear_up);
```

Comment:
These last two lines aren't needed: this can simply return `base+variable`.  (`linear_up` is never going to be the larger term in the max since this now only runs for `height < 235987`, so the calculation and the max is just dead code.)

Reply:
Nice catch .. this will be updated shortly